user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+the Outlook sidebar.
+
+This is a general capability that can improve the user experience for all Enterprise users who are using both OroCRM and 
+Outlook. 
+
+
+|
+
+.. image:: ./img/intro/user_outlook.png
+
+|
+
+
+.. _admin-capabilities-address-dic:
+
+Read Address Dictionaries
+-------------------------
+
+If the capability is set to *"System"*, the user can access countries, regions, and address types via the API. 
+It has to be set to *"System"* in order to support Lead creation via Outlook. This capability should be activated to 
+system administrators or integrators who are authorized to access OroCRM via the API.
+
+
+.. _admin-capabilities-search:
+
+Search
+------
+
+If the capability is set to *"System"*, the user can use the :ref:`search <user-guide-getting-started-search>` 
+functionality to quickly find specific records.
+
+This is a general capability that can improve the overall experience of all users.
+
+The setting does not influence the user's ability to :ref:`search by tag <user-guide-getting-started-search-tag>`.
+
+
+.. _admin-capabilities-campaign-emails:
+
+Send Campaign Emails
+--------------------
+
+With OroCRM's :ref:`email campaigns <user-guide-email-campaigns>`, the user can send personalized template-based emails 
+to multiple users. This capability doesn't effect the users' ability to define and edit the campaign settings and create 
+templaets, but on if this capability is set to *"System"*, the user can launch the camplaign, i.e. start 
+:ref:`sending emails <user-guide-email-campaigns-send>` specified by the campaign.
+
+Marketing associates and other employees authorized for direct communications with potential 
+customers, existing clients, other system users, etc. are usually authorized to send email campaigns.
+
+|
+
+.. image:: ./img/roles/email_campaign.png
+
+|
+
+
+.. _admin-capabilities-share-grid:
+
+Share Grid View
+---------------
+
+If this capability is set to *"System"*, the user can share the :ref:`grid <user-guide-ui-components-grids>` views 
+that they have configured, this way the user can :ref:`adjust a grid <user-guide-ui-components-grids-adjust>` and share 
+it with other users.
+
+This is particularly useful for team-leads and heads of departments, to modify and share grids with their subordinates. 
+
+|
+
+.. image:: ./img/roles/grid_share.png
+
+|
+
+ 
+.. _admin-capabilities-system-info:
+
+System Information
+------------------
+
+If this capability is set to *"System"*, the user can view the system information page. This page contains the list of 
+Oro packages and third-party packages that are installed, and is usually only used by system administrators and 
+integrators.
+
+.. _admin-capabilities-system-config:
+
+System Configuration
+--------------------
+
+If this capability is set to *"System"*, the user can access the :ref:`system configuration page <admin-configuration>`
```

Comment:
Edit: If this capability is set to *"System,"* the user can access the :ref:`system configuration page <admin-configuration>`